ngraph_bridge/ngraph_builder.cc

```diff
@@ -2367,25 +2272,10 @@ static Status TranslateGatherV2Op(
                                    "), but got ", tf_axis[0]);
   }
 
-  auto config_map = BackendManager::GetBackendAttributeValues(backend_name);
-  if (config_map.at("ngraph_backend") != "NNPI") {
-    auto gather_op = ConstructNgNode<ng::op::Gather>(
-        op->name(), ng_input, ng_input_coords, tf_axis[0]);
-
-    SaveNgOp(ng_op_map, op->name(), gather_op);
-  } else {
-    ng::runtime::Backend* backend = BackendManager::GetBackend(backend_name);
-
-    shared_ptr<ng::Node> ng_gather =
-        backend->get_backend_op("Gather", &ng_input, &ng_input_coords, &axis);
-    if (ng_gather == nullptr) {
-      return errors::Internal("In translating GatherV2 op ", op->name(),
-                              " backend could not return valid ngraph node");
-    }
-    Builder::SetTracingInfo(op->name(), ng_gather);
-    SaveNgOp(ng_op_map, op->name(), ng_gather);
-  }
+  auto gather_op = ConstructNgNode<ng::op::Gather>(
```

Comment:
Shouldn't this be opset3?

Reply:
The intent of this PR was to primarily remove backend-specific ops, not upgrade any ops. We'll address upgrade of all Gather ops in a separate PR.